Tiltfile

```diff
@@ -160,6 +171,12 @@ def observability():
     )
     k8s_resource(workload = "opentelemetry-collector", labels = ["observability"])
     k8s_resource(workload = "opentelemetry-collector-agent", labels = ["observability"])
+    k8s_resource(
+        workload = "capi-visualizer",
+        new_name = "visualize-cluster",
+        port_forwards = [port_forward(local_port = 8000, container_port = 8081, name = "View visualization")],
```

Comment:
I wonder if we should use another local port to leave 8000 free?

Reply:
Yeah I was wondering if we had any convention for which ports we used. It looks like Jaeger is on port 16686 and Prometheus is on port 9090. Is there any particular reason for that?

Reply:
I think I misread, saw 8000 and 8081 and converged them to 8080 in my head. 

8000 should be fine